### PR TITLE
blacklist linux-aws versions prior to 4.4.0-1060

### PR DESCRIPTION
**Pull request by DylanLovesCoffee**
### What does this PR do?

linux-aws kernel 4.4.0-1060.69 holds the patch for this bug: https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1763454. Any linux-aws kernel version that precedes will not be supported.

### Motivation

This was brought to our attention by a customer.

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: pkg/ebpf/common_test.go [hunk 1]
```diff
@@ -25,6 +25,14 @@ func TestUbuntu44119NotSupported(t *testing.T) {
 	}
 }
 
+func TestLinuxAWSPreceding441060NotSupported(t *testing.T) {
+	for i := uint32(120); i < 128; i++ {
+		ok, err := verifyOSVersion(linuxKernelVersionCode(4, 4, i), "Linux-4.4.0-1060-aws", nil)
+		assert.False(t, ok)
+		assert.NotEmpty(t, err)
```
Comment (kevinconaway) [inline pkg/ebpf/common_test.go]: FYI there is an `assert.Error()` func for this.

**Review by kevinconaway — DISMISSED** [review]

File: pkg/ebpf/common_test.go (hunk 1 above)
Comment (shang-wang) [reply pkg/ebpf/common_test.go]: Does `assert.Error()` work with `string`?

Comment (shang-wang) [reply pkg/ebpf/common_test.go]: To be fair, the variable `err` here is misleading, it should be called `message` or something instead, in golang `err` usually represents type `error`.

Comment (DylanLovesCoffee) [reply pkg/ebpf/common_test.go]: Makes sense - and good catch 💯 I'll change the involved tests accordingly

**Review by shang-wang — DISMISSED** [review]

File: pkg/ebpf/common.go [hunk 2]
```diff
@@ -89,6 +89,10 @@ func verifyOSVersion(kernelCode uint32, platform string, exclusionList []string)
 		if kernelCode >= linuxKernelVersionCode(4, 4, 119) && kernelCode <= linuxKernelVersionCode(4, 4, 126) {
 			return false, fmt.Sprintf("got ubuntu kernel %s with known bug on platform: %s, see: https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1763454", kernelCodeToString(kernelCode), platform)
 		}
+	} else if isLinuxAWS(platform) {
+		if kernelCode < linuxKernelVersionCode(4, 4, 128) {
```
Comment (leeavital) [inline pkg/ebpf/common.go]: How do you get 128 from `kernel 4.4.0-1060.69` ?

Comment (DylanLovesCoffee) [reply pkg/ebpf/common.go]: When we call [`CurrentKernelVersion()`](https://github.com/iovisor/gobpf/blob/362e99fa21988d6b1eecbb3aa683e55a4a622029/elf/kernel_version.go#L103) it ends up returning the release string at the end of the version signature for Ubuntu. So when I ran this in an EC2 it resolved to `4.4.128`, with `128` being the kernel patch number:

```
ubuntu@ip-###-##-###-##:~$ cat /proc/version_signature
Ubuntu 4.4.0-1060.69-aws 4.4.128
```

**Review by leeavital — APPROVED** [review]

**Pull request merged**